api/controllers/employees.js

```diff
@@ -0,0 +1,86 @@
+const sequelize = require('sequelize');
```

Comment:
let's not use grouping based on the type of a code structure. Practice shows that a grouping files in folders like `controllers` has many issues when number of controllers grows significantly. Better alternative is when a controller resides closer to what it "controls" e.g. the router (there might be other options)

Reply:
what structure can you suggest for routes controllers/controls here? (how group)